### DISCUSSION TITLE
求助：有没有办法一键进入Win10的蓝牙连接界面

### DISCUSSION THREAD

**Discussion by modyson** [shuax/MouseInc.Settings#278] [Q&A] [answered]
因为经常要切换蓝牙耳机或蓝牙音响，一直没找到快捷键。一般是用 Win + X + N ，然后再点击“设备”选项，进入蓝牙连接界面的。想问问大神，有没有什么快捷键可以配置成手势，一键进入。

**Comment by 6Tom** [comment]
打开设置可以用win+i，帮您省了一个键位，惊喜不😂

**Reply by modyson** [reply]
你这个方法也不错

**Comment by shuax** [comment, marked as answer]
试试运行 ms-settings:bluetooth

**Reply by modyson** [reply]
神了，果然是大神！

**Comment by 08spider** [comment]
win+k  更快
